README.md

```diff
@@ -100,6 +100,70 @@ resources:
 Don't forget, you'll need to mount the directory containing the unix socket that
 the KMS server is listening on into the kube-apiserver.
 
+### Bootstrap during cluster creation (kops)
+To use encryption provider during cluster creation, you need to ensure that its running
+before starting kube-apiserver. For that you need to follow following high level steps.
+
+Note: These steps have been verified with [kops](https://github.com/kubernetes/kops) but 
+it should be similar to any other cluster bootstrapping tool.
+
+#### Run aws-encryption-provider as static pod
+You need to have encryption provider running before kube-apiserver, and to do that you can
+use [static pods](https://kubernetes.io/docs/tasks/administer-cluster/static-pod/) functionality. For kops, static pod manifests are available at `/etc/kubernetes/manifests`. You can further use kops file assets functionality to drop 
```

Comment:
nit: Did you mean to wrap this like at 90 like the rest?

Reply:
i noticed it, but didn't do it, as in actual readme preview, this will only take 11 chars as oppose to 79 chars in raw readme. 

please refer to: https://github.com/rajatjindal/aws-encryption-provider/tree/update-readme

what do you think?

Reply:
no changes required. Just curious